From 8d86b55ae37155228d45c9a6812a38f76d95d974 Mon Sep 17 00:00:00 2001
From: Sandor Yu <R01008@freescale.com>
Date: Fri, 20 Mar 2015 11:27:10 +0800
Subject: [PATCH] MLK-10447-03: ARM: imx_v7_defconfig: Add spi gpio and gpio
 74x164

Enable CONFIG_SPI_GPIO and CONFIG_GPIO_74X164 to imx_v7_defconfig.

Signed-off-by: Sandor Yu <R01008@freescale.com>
---
 arch/arm/configs/imx_v7_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/configs/imx_v7_defconfig b/arch/arm/configs/imx_v7_defconfig
index ebbce5f..21cd4ec 100644
--- a/arch/arm/configs/imx_v7_defconfig
+++ b/arch/arm/configs/imx_v7_defconfig
@@ -183,8 +183,10 @@ CONFIG_I2C_ALGOPCF=m
 CONFIG_I2C_ALGOPCA=m
 CONFIG_I2C_IMX=y
 CONFIG_SPI=y
+CONFIG_SPI_GPIO=y
 CONFIG_SPI_IMX=y
 CONFIG_GPIO_SYSFS=y
+CONFIG_GPIO_74X164=y
 CONFIG_POWER_SUPPLY=y
 CONFIG_SABRESD_MAX8903=y
 CONFIG_IMX6_USB_CHARGER=y
-- 
1.8.0

